pom.xml

```diff
@@ -349,6 +349,16 @@
             <artifactId>google-cloud-storage</artifactId>
             <version>1.2.1</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-firestore</artifactId>
+            <version>0.25.0-beta</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.api</groupId>
+            <artifactId>gax</artifactId>
```

Comment:
Why do we need this?  I'm not really sure what it is, but I don't see any obvious references to it (e.g. com.google.api classes or anything)...

Reply:
This is a transitive dependency for Firestore/GRPC. Without it I get the following test failures:

```
testServiceAccountProjectId(com.google.firebase.cloud.FirestoreClientTest)  Time elapsed: 0.001 sec  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/api/gax/rpc/ClientSettings
	at com.google.firebase.cloud.FirestoreClientTest.testServiceAccountProjectId(FirestoreClientTest.java:42)
Caused by: java.lang.ClassNotFoundException: com.google.api.gax.rpc.ClientSettings
	at com.google.firebase.cloud.FirestoreClientTest.testServiceAccountProjectId(FirestoreClientTest.java:42)

testAppDelete(com.google.firebase.cloud.FirestoreClientTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.google.cloud.firestore.FirestoreOptions
	at com.google.firebase.cloud.FirestoreClientTest.testAppDelete(FirestoreClientTest.java:56)

testExplicitProjectId(com.google.firebase.cloud.FirestoreClientTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class com.google.cloud.firestore.FirestoreOptions
	at com.google.firebase.cloud.FirestoreClientTest.testExplicitProjectId(FirestoreClientTest.java:30)
```

I think Firestore should declare this dependency. I'll report this for Firestore.

Reply:
Yeah, I don't think that should be necessary.  FWIW, our docs don't mention needing it (https://firebase.google.com/docs/firestore/quickstart under the java tab just shows the one import)

Reply:
Reported at https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2496